src/main/java/edu/kit/hci/soli/controller/BookingsController.java

```diff
@@ -150,7 +158,7 @@ public String newBooking(
     @PostMapping(value = "/{roomId}/bookings/new", consumes = "application/x-www-form-urlencoded")
     public String createBooking(
             Model model, HttpServletResponse response, HttpServletRequest request, @PathVariable Long roomId,
-            @AuthenticationPrincipal User user,
+            @AuthenticationPrincipal SoliUserDetails user,
```

Comment:
I think we should rename these variables to soliUserDetails...  It's not really the user but rather a wrapper

Reply:
True. I think I'll go with `principal` since that represents its actual function